product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -65,6 +65,7 @@ class ProductPricelistPrint(models.TransientModel):
     lang = fields.Selection(
         _lang_get, string="Language", default=lambda self: self.env.user.lang
     )
+    products_from_date = fields.Datetime("Products ordered from date")
```

Comment:
The field name is a bit misleading (and I always try to avoid plurals). What about?

```suggestion
    product_selling_date_threshold = fields.Datetime(string="Selling date threshold", help="Filter only the products ordered since this date")
```